src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="container">
+  <hr />
+
+  <form class="form">
+    <div>
+      <h3 class="form__headline">Інформація про клієнта</h3>
+    </div>
+    <div class="form__wrapper row d-flex">
+      <div class="form__block col-md-4 col-sm-6">
+        <div class="form-group">
+          <p class="form__subtitle">Клієнт</p>
+          <label class="form__label" for="name">Ім'я</label>
+          <input class="form-control" type="text" id="name" placeholder="Placeholder" />
+        </div>
+        <div class="form-group">
+          <label class="form__label" for="name">Телефон</label>
+          <input class="form-control" type="text" id="tel" placeholder="Placeholder" />
+        </div>
+        <div class="form-group">
+          <label class="form__label" for="name">Email</label>
+          <input class="form-control" type="text" id="email" placeholder="Placeholder" />
+        </div>
+      </div>
+
+      <div class="form__block col-md-4 col-sm-6 offset-md-1">
+        <div class="form-group">
+          <p class="form__subtitle">Отримувач</p>
+          <label class="form__label" for="name">Ім'я</label>
+          <input class="form-control" type="text" id="name" placeholder="Placeholder" />
+        </div>
+        <div class="form-group">
+          <label class="form__label" for="name">Телефон</label>
+          <input class="form-control" type="text" id="tel" placeholder="Placeholder" />
```

Comment:
Change 'for' attribute from 'name' to 'tel'

Reply:
✅